pkg/integrations/manager.go

```diff
@@ -231,18 +238,27 @@ func (m *Manager) instanceConfigForIntegration(cfg Config, i Integration) instan
 	relabelConfigs := append(m.defaultRelabelConfigs, common.RelabelConfigs...)
 
 	var scrapeConfigs []*config.ScrapeConfig
+	schema := "http"
+	// Check for HTTPS support
+	httpClientConfig := config_util.HTTPClientConfig{}
```

Comment:
For zero values, use the `var` form instead of `:=`:

```suggestion
	var httpClientConfig config_util.HTTPClientConfig
```

Link to the [style guide](https://github.com/uber-go/guide/blob/master/style.md#local-variable-declarations) we've passed around internally

Reply:
Done and noted